docs/integrations/files/parquet.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Upload Parquet files to MindsDB
+sidebarTitle: Parquet
+---
+
+You can upload Parquet files of up to 500MB in size to MindsDB.
```

Comment:
Maybe we should mention that the 500MB is a limit in our SQL Editor? Same for all other file types

Reply:
Ok. Now I'm wondering why do we set this limit in the GUI -- is there any reason for that?

I added this to docs:
```
 Note that this limit is imposed when uploading a file via MindsDB SQL Editor. If you [upload a file via APIs](https://docs.mindsdb.com/rest/files/upload), this limit does not persist.
```